dojo/tools/anchore_grype/parser.py

```diff
@@ -23,44 +22,143 @@ def get_findings(self, file, test):
         data = json.load(file)
         dupes = dict()
         for item in data.get("matches", []):
-            cve = item["vulnerability"]["id"]
-            severity = self._convert_severity(item["vulnerability"]["severity"])
-            purl = PackageURL.from_string(item["artifact"]["purl"])
-            description = ""
-            description += f"\n**CVE:** {cve}"
-            if type(item["matchDetails"]) is dict:
-                description += f'\n**Matcher:** {item["matchDetails"]["matcher"]}'
+            vulnerability = item['vulnerability']
+            vuln_id = vulnerability["id"]
+            vuln_namespace = vulnerability.get('namespace', None)
+            vuln_datasource = vulnerability.get('dataSource', None)
+            vuln_severity = self._convert_severity(vulnerability["severity"])
+            vuln_urls = vulnerability.get('urls', None)
+            vuln_description = vulnerability.get('description', None)
+            vuln_fix_versions = None
+            fix = vulnerability.get('fix', None)
+            if fix:
+                vuln_fix_versions = fix.get('versions', None)
+            vuln_cvss = vulnerability.get('cvss', None)
+
+            rel_id = None
+            rel_datasource = None
+            rel_urls = None
+            rel_description = None
+            rel_cvss = None
+            related_vulnerabilities = item.get('relatedVulnerabilities', None)
+            if related_vulnerabilities:
+                related_vulnerability = related_vulnerabilities[0]
+                rel_id = related_vulnerability.get('id')
+                rel_datasource = related_vulnerability.get('dataSource', None)
+                rel_urls = related_vulnerability.get('urls', None)
+                rel_description = related_vulnerability.get('description', None)
+                rel_cvss = related_vulnerability.get('cvss', None)
+
+            matches = item['matchDetails']
+
+            artifact = item['artifact']
+            artifact_name = artifact.get('name', None)
+            artifact_version = artifact.get('version', None)
+            artifact_purl = artifact.get('purl', None)
```

Comment:
```suggestion
```
If you want the parser to not use anymore PURL data, just remove it completely.

Reply:
The `purl` is still used as part of the description.